examples/plugins/zabbix/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "suricata-zabbix"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib"]
```

Comment:
I think this is correct for using the C ABI?  @chifflier?

Reply:
@chifflier this may interest you https://github.com/OISF/suricata/pull/12062